assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -919,9 +918,19 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
     if (!simLife.init) {
       sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.INITIAL_SETUP_COMPLETE, {});
       simLife.init = true;
+      if (screenContext === NotificationType.CONTEXT_CHANGED) {
```

Comment:
for bind To variables, the SIM expects a value change request after we have completed the initial setup. This value is considered as the base value by that SIM. When a new values is received, that gets displayed in the textbox